### ISSUE TITLE
HTTP Basic Auth: Impossible to fill credentials with Bitwarden

### ISSUE THREAD

**Issue by immanuelfodor** [brave/browser-android-tabs#2411]
## Description 

The HTTP Basic Auth dialog is impossible to fill with Bitwarden. (Probably with any password manager that selects the entry in a new window then takes back to the previos app.)

## Steps to reproduce 

1. Have the latest Brave Android and Bitwarden installed on your phone
2. Open a site with HTTP Basic Auth in Brave
3. Tap into the password field, Bitwarden suggests auto fill
4. Tap the auto fill notification, Bitwarden loads and lets you select an entry
5. Tap on the selected entry, choose auto fill, then you're taken back to the previous app
6. The HTTP Basic Auth dialog is still displayed but closes in an instant, and a HTTP 401 error is displayed from the underlying website

## Actual result 

The HTTP Basic Auth dialog closes before auto fill could happen, and so the browser can't send the credentials, which result in an unauthorized site access. The only way to fill a HTTP Basic Auth dialog with Brave Android is to copy the password to clipboad beforehand (unsafe!) and paste it into the input field.

## Expected result

The HTTP Basic Auth dialog stays open until the user taps okay/login. Firefox Android implements this nicely, only user interaction closes the dialog after the auto fill.

## Issue reproduces how often 

Every time with Brave Android, for months.

## Issue happens on 
   - Current Play Store version? Yes
   - Beta build? No (did not try)

## Device details
   - Install type (ARM, x86): arm64
   - Device (Phone, Tablet, Phablet): OnePlus 7
   - Android version: Oxygen OS 10.0.3 (latest)
   
## Brave version

From `chrome://version`: 79.0.3945.93 (Official Build) (32-bit)

## Additional information

This is not a new issue, it has been present since I know (for months), I've just found out that Firefox is exempt from this bug, so it must be a Brave/Chromium specific issue.

**Comment by immanuelfodor** [comment]
The same happens if you have the dialog open, then switch to another app, then back to Brave. So it might be not Bitwarden specific as well. Firefox keeps the dialog open between app switches, that's why it works.

**Comment by srirambv** [comment]
Can you confirm if this happens on Chrome? It may need an upstream fix

**Comment by immanuelfodor** [comment]
Yes, it's the same with Chrome. Could you please eccalate the issue to them if Brave can't be fixed on its own?
